From 9e109a260083078a51a5b76fddcd2dcc3b298bfb Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Thu, 28 May 2020 09:19:15 +0800
Subject: [PATCH] ARM: dts: rockchip: reserve 32KB shared memory for
 rv1126-evb-ddr3-v10-tb

Reserve shared memory between Trust and Linux.

Change-Id: Ifb06c758f6e19d2c79fb3047a8ae2787576801f1
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts
index b7e66776b11f..1380248a2feb 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts
@@ -25,7 +25,7 @@
 
 	reserved-memory {
 		trust@0 {
-			reg = <0x00000000 0x00200000>;
+			reg = <0x00000000 0x00208000>;
 			no-map;
 		};
 
-- 
2.35.3

